### ISSUE TITLE
tests fail with tensorflow 10.0rc0

### ISSUE THREAD

**Issue by jameshensman** [GPflow/GPflow#163]
One test fails with the new tensorflow pre-release, it's related to the custom-op test:
```python
ImportError: No module named 'tensorflow.python.kernel_tests'
```

In addition, there sees to be an issue with one of the notebooks ( #161 ), I'm looking into this.

**Comment by alexggmatthews** [comment]
I've seen the same bug. It looks to be only with more recent versions of Tensorflow and is localised to the gradient of reduce_prod which has recently changed in tf. It is therefore either a bug with TensorFlow or has revealed something about the shape of the GPflow tensors thatt it was previously willing to tolerate. My guess is its a bug with TensorFlow.

**Comment by markvdw** [comment]
Fixed in PR #165.
